.github/workflows/benchmarks.yml

```
@@ -62,11 +62,8 @@ jobs:
       # Download cached remote files (artifacts) from GitHub
       - name: Download remote data from GitHub
         run: |
-          gh run download -n gmt-cache -D gmt-cache
-          # Move downloaded files to ~/.gmt directory and list them
-          mkdir -p ~/.gmt
-          mv gmt-cache/* ~/.gmt
-          rmdir gmt-cache
+          # Download cached files to ~/.gmt directory and list them
+          gh run download -n gmt-cache -D ~/.gmt/
```

Use long-form for readability
```suggestion
          gh run download --name gmt-cache --dir ~/.gmt/
```